wasm/wasm_source/src/tx_enqueue_eth_transfer.rs

```diff
@@ -0,0 +1,41 @@
+use anoma_tx_prelude::{
+    log_string, transaction, write, BorshDeserialize, Signed,
+};
+
+const TX_NAME: &str = "tx_enqueue_eth_transfer";
+
+const ETH_BRIDGE_QUEUE_STORAGE_KEY: &str = "#atest1v9hx7w36g42ysgzzwf5kgem9ypqkgerjv4ehxgpqyqszqgpqyqszqgpqyqszqgpqyqszqgpq8f99ew/queue";
+
+fn log(msg: &str) {
+    log_string(format!("[{}] {}", TX_NAME, msg))
+}
+
+fn fatal(msg: &str, err: impl std::error::Error) -> ! {
+    log(&format!("ERROR: {} - {:?}", msg, err));
+    panic!()
+}
+
+fn fatal_msg(msg: &str) -> ! {
+    log(msg);
+    panic!()
+}
```

Comment:
I had to put a fair bit of logging while developing and unsure how much to remove, I guess it would add to gas cost